Mirror/Runtime/NetworkClient.cs

```diff
@@ -194,11 +186,9 @@ internal virtual void Update()
                 return;
             }
 
-            if (connectState == ConnectState.Connected && Time.time - lastPingTime >= pingFrequency)
+            if (connectState == ConnectState.Connected)
             {
-                NetworkPingMessage pingMessage = NetworkTime.GetPing();
-                Send((short)MsgType.Ping, pingMessage);
-                lastPingTime = Time.time;
+                NetworkTime.SendPing(this);
```

Comment:
this might be confusing. it looks like it sends a ping each Update. is there a better way or a better naming?
maybe do NetworkTime.Update() and pass the connectState or do if connected: Update()?

Reply:
if connectstate==connected: NetworkTime.Update() might actually be a good idea.
no need to update if not connected.
and having Update() in there allows us to add more update-like features to networktime if needed some day

Reply:
yes,  very good points.   Updated.